packages/nuxt/pages/playground.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="playground" style="padding: 1rem">
+    <SfPagination :total="10"/>
+  </div>
+</template>
+<script>
+  import { SfImage, SfPagination } from "@storefront-ui/vue"
+  export default {
+    name: "playground",
+    layout: "blank",
+    components: { SfImage, SfPagination },
+    data(){
+      return {
+        src: "/assets/storybook/SfImage/product-216x326.jpg",
+        ob: {
+          mobile: {
+            url: "https://storybook.storefrontui.io/assets/storybook/SfImage/product-109x164.jpg"
```

Comment:
I think we do not use `https://storybook.storefrontui.io` anymore

Reply:
I'll change that.